lib/elixir_analyzer/test_suite/newsletter.ex

```diff
@@ -20,9 +20,24 @@ defmodule ElixirAnalyzer.TestSuite.Newsletter do
     end
   end
 
+  feature "use IO.puts in log_sent_email rather than IO.write" do
+    find :none
+    type :actionable
+    comment Constants.newsletter_log_sent_email_prefer_io_puts()
+
+    form do
+      def log_sent_email(_ignore, _ignore) do
+        _block_ends_with do
```

Comment:
I think we don't need `_block_ends_with` because we would miss a solution like
```
ok =  IO.write(file, line <> "\n")
ok
```
Maybe `assert_no_call` is more appropriate for this particulate check.

Reply:
Hmm. I did that on purpose so that solutions that use `IO.write` and don't end with `IO.write` will be pointed to this comment instead which also mentions `IO.puts`: https://github.com/exercism/website-copy/blob/main/analyzer-comments/elixir/newsletter/log_sent_email_returns_implicitly.md

I could add a separate check that will trigger both comments in this case (this one needs to stay like that to correctly suppress the no implicit return check)

Reply:
In that case, we would be back to the misleading error that lead me to open the issue. 
Keeping this particular check focused on puts vs. write feels better to me. 

How about removing the `suppress_if`? In this case no matter what, if they use `IO.write`, they get 2 comments: one saying use `IO.puts` and one saying put `IO.puts` at the end. Reading those 2 in a row would be fine.

Reply:
Actually I've re-read your suggestion and yes, it's probably better than mine.

Reply:
Done: [`68b7f74` (#167)](https://github.com/exercism/elixir-analyzer/pull/167/commits/68b7f74ca525239021fb41969e26275d44a6a157)